pjsip/include/pjsip/sip_parser.h

```diff
@@ -417,6 +418,10 @@ PJ_DECL(void) pjsip_parse_end_hdr_imp ( pj_scanner *scanner );
 PJ_DECL(void) pjsip_parse_generic_array_hdr_imp(pjsip_generic_array_hdr *hdr,
 						pj_scanner *scanner);
 
+/* Parse generic array header with arbitrary delimiter */
+PJ_DECL(void) pjsip_parse_generic_delimited_array_hdr(pjsip_generic_array_hdr *hdr,
+                        pj_scanner *scanner,char delimiter, const pj_cis_t *not_delimiter_or_newline);
```

Comment:
not_delimiter_or_newline should be of type `const pj_cis_t* const` – see https://isocpp.org/wiki/faq/const-correctness#const-ptr-vs-ptr-const